packages/js/client/README.md

```diff
@@ -7,6 +7,8 @@
 <br/>
 The Polywrap JavaScript client invokes functions of wrappers and plugins. It's designed to run in any environment that can execute JavaScript (think websites, node scripts, etc.). It has TypeScript support.
```

Comment:
should this paragraph be moved to `core-client`? the reasoning is that this package doesn't take the responsibility of the invocation (or any), it just exists to have an ez way to config the client, right?